### ISSUE TITLE
Last release 0.4.0 is not in helm repo index

### ISSUE THREAD

**Issue by librucha** [passbolt/charts-passbolt#34]
If i tried to search latest relased version of helm chart 0.4.0 i found naothing in index

```bash
curl -s  https://download.passbolt.com/charts/passbolt/index.yaml | yq '.entries.passbolt[] | select(.version | test("0.4.0"))'
```
Please add it to the index.

**Comment by dlen** [comment]
Thanks for the heads up! It should be fixed now!

**Comment by librucha** [comment]
Great. It’s working well now.

BR
Libor Ondrušek /IT

+420 605 769 041
***@***.******@***.***> – www.rixo.cz<https://www.rixo.cz/>
Pikrtova 1a, 140 00  Praha 4 – ENTERPRISE OFFICE CENTER


[image001.png]

 [image002.png]    <https://www.instagram.com/rixo_cz/> [image003.png]    <https://www.linkedin.com/company/rixocz/> [image004.png]

On 14. 7. 2023, at 14:54, Diego Lendoiro ***@***.***> wrote:



Closed #34<https://github.com/passbolt/charts-passbolt/issues/34> as completed.
